src/js/conntest.js

```diff
@@ -7,100 +7,64 @@
  */
 'use strict';
 
-addTest(testSuiteName.CONNECTIVITY, testCaseName.UDPCONNECTIVITY,
-    udpConnectivityTest);
-addTest(testSuiteName.CONNECTIVITY, testCaseName.TCPCONNECTIVITY,
-    tcpConnectivityTest);
-addTest(testSuiteName.CONNECTIVITY, testCaseName.IPV6ENABLED,
-    hasIpv6CandidatesTest);
+addTest(testSuiteName.CONNECTIVITY, testCaseName.RELAYCONNECTIVITY,
+    relayConnectivityTest);
+addTest(testSuiteName.CONNECTIVITY, testCaseName.REFLEXIVECONNECTIVITY,
+    reflexiveConnectivityTest);
+addTest(testSuiteName.CONNECTIVITY, testCaseName.HOSTCONNECTIVITY,
+    hostConnectivityTest);
 
-// Test whether it can connect via UDP to a TURN server
-// Get a TURN config, and try to get a relay candidate using UDP.
-function udpConnectivityTest() {
-  Call.asyncCreateTurnConfig(
-      function(config) {
-        filterConfig(config, 'udp');
-        gatherCandidates(config, null, Call.isRelay);
-      },
-      reportFatal);
-}
+var timeout = null;
 
-// Test whether it can connect via TCP to a TURN server
-// Get a TURN config, and try to get a relay candidate using TCP.
-function tcpConnectivityTest() {
+function relayConnectivityTest() {
   Call.asyncCreateTurnConfig(
-      function(config) {
-        filterConfig(config, 'tcp');
-        gatherCandidates(config, null, Call.isRelay);
-      },
-      reportFatal);
-}
-
-// Test whether it is IPv6 enabled (TODO: test IPv6 to a destination).
-// Turn on IPv6, and try to get an IPv6 host candidate.
-function hasIpv6CandidatesTest() {
-  var params = {optional: [{googIPv6: true}]};
-  gatherCandidates(null, params, Call.isIpv6);
+    runConnectivityTest.bind(null, Call.isRelay),
+    reportFatal);
```

Comment:
But it does not care here https://github.com/webrtc/testrtc/pull/127/files#diff-4d6dd141efeb8b75cad3c691015422f8R91